openslides_backend/action/action.py

```diff
@@ -181,6 +183,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         """
         return instance
 
+    def final_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
use augment/modify instead of update since it is expected to return the instance.

Reply:
It is named (final_)update_instance, because `update_instance` is the name of the method working the same way, which is called at the beginning  of the action process.